test/client/api.js

```diff
@@ -153,6 +155,49 @@ ClientApi.prototype.accountKeys = function (keyFetchTokenHex) {
     )
 }
 
+ClientApi.prototype.accountDevices = function (sessionTokenHex) {
+  return tokens.SessionToken.fromHex(sessionTokenHex)
+    .then(
+      function (token) {
+        return this.doRequest(
+          'GET',
+          this.baseURL + '/account/devices',
+          token
+        )
+      }.bind(this)
+    )
+}
+
+ClientApi.prototype.updateDevice = function (sessionTokenHex, info) {
+  return tokens.SessionToken.fromHex(sessionTokenHex)
+    .then(
+      function (token) {
+        return this.doRequest(
+          'POST',
+          this.baseURL + '/account/device',
+          token,
+          info
+        )
+      }.bind(this)
+    )
+}
+
+ClientApi.prototype.deleteDevice = function (sessionTokenHex, id) {
```

Comment:
Naming nit: the other methods in this file seem to be "nounVerb", like "accountDestroy" and "passwordChangeStart".  For consistency, should we name these something like "deviceUpdate" and "deviceDestroy"?